ts/protocol/errors.ts

```diff
@@ -107,6 +108,13 @@ export class OpenYoloError {
         {code: ERROR_TYPES.illegalState, message: reason});
   }
 
+  static illegalConcurrentRequestError() {
+    return OpenYoloError.createError({
+      code: ERROR_TYPES.illegalConcurrentRequest,
+      message: 'Concurrent requests are not authorized'
```

Comment:
The phrasing is a little strange here, particularly for an auth product. Replace "authorized" with "permitted".